<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ssp</class>
 <widget class="QWidget" name="ssp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ssp</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,3">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetLeft" native="true">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,1">
      <item row="0" column="0">
       <widget class="QWidget" name="widgetLoad" native="true">
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,1,1,1">
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonNew">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButtonOld">
           <property name="text">
            <string>Old</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="labelLoad">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widgetAlgorithm" native="true">
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,1,1,1">
         <item row="1" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButtonFloyd">
           <property name="text">
            <string>Floyd</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="radioButtonDijkstra">
           <property name="text">
            <string>Dijkstra</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="labelAlgorithm">
           <property name="text">
            <string>Algorithm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widgetPoint" native="true">
        <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1,1,1,2">
         <item row="1" column="0">
          <widget class="QLabel" name="labelStart">
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEditStart"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelEnd">
           <property name="text">
            <string>End:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="lineEditEnd"/>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QPushButton" name="pushButtonCalculate">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="labelPoint">
           <property name="text">
            <string>Point</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widgetMethod" native="true">
        <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,1,1,1,1">
         <item row="1" column="3">
          <widget class="QRadioButton" name="radioButtonTime">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButtonDistance">
           <property name="text">
            <string>Distance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="labelMethod">
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetRight" native="true">
     <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,20">
      <item row="0" column="0">
       <widget class="QLabel" name="labelResult">
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widgetResult" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
